source/reference/minio-mc-admin/mc-admin-user-svcacct.rst

```diff
@@ -87,6 +87,8 @@ Syntax
                --policy "/path/to/policy.json"              \
                myminio myuser
 
+         The command returns an output of the access key and secret key for the new account.
```

Comment:
Is `an output of` significant here, or could this be `returns the access key...` instead?

Reply:
Not significant, cut.